src/Integration.Vsix.UnitTests/Commands/ProjectTestPropertySetCommandTests.cs

```diff
@@ -19,9 +19,9 @@
  */
```

Comment:
Lots of tests in here are identical to the one in the PR -> https://github.com/SonarSource/sonarlint-visualstudio/pull/4828

I'll leave a pr comment wherever this applies